### PR TITLE
RDPP-5798: Remove unnecessary fluff and fix audit issues

### PR DESCRIPTION
**Pull request by alasdairhurst**
Removed unused dependencies causing security audit issues. Removed lib from git (generated in build) and removed sourcemaps (meh)

Removed test and other dev files from published files.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,29 +24,26 @@
...
-        "lodash": "^3.10.1",
-        "pkginfo": "^0.3.1",
-        "semver": "^5.0.3",
-        "cookies": "^0.5.1",
         "passport": "^0.3.2",
         "passport-saml": "^0.31.0"
     },
     "devDependencies": {
-        "arrow": "^*",
+        "arrow": "rdpp-v3staging",
```
Comment (wickedest) [inline package.json]: are you intentionally using an rdpp branch?

**Comment by wickedest** [comment]
empty file: conf/.npmignore

File: package.json (hunk 1 above)
Comment (alasdairhurst) [reply package.json]: Ah this was for testing what dependencies got pulled. i'll revert this back

Comment (alasdairhurst) [reply package.json]: Pointed to ^3.0.0 instead. Better than using * but in reality, i doubt it will ever be used again.

**Pull request merged**